src/canopy_cli/cli.py

```diff
@@ -102,7 +102,9 @@ def _initialize_tokenizer():
 
 def _read_config_file(config_file: Optional[str]) -> Dict[str, Any]:
     if config_file is None:
-        return {}
+        config_file = os.getenv("CANOPY_CONFIG_FILE", None)
```

Comment:
I believe `click` has a built-in solution, that should also automatically appear in the help text.  
If you'll look at the click declaration of the `--index-name` argument of `canopy new`, you'd notice an `envvar=` option that that automatically sets that argument to also be read from an en var.